README.md

```diff
@@ -20,19 +20,40 @@ module.exports = {
     {
       resolve: `gatsby-plugin-material-ui`,
       // If you want to use styled components, in conjunction to Material-UI, you should:
-      // - Change the injection order
+      // - Change the injection order by adding a gatsby-mui-config.js and specifying the path in pluginConfigModule (e.g. src/utils/gatsby-mui-config.js)
       // - Add the plugin
+      // Use either injectFirst or customInPoint (not both)
       options: {
-        // stylesProvider: {
-        //   injectFirst: true,
-        // },
+        injectStyles: true,
```

Comment:
What is injectStyles? Did you mean `injectFirst`?

Reply:
Hi @oliviertassinari . Nope, it is an option that I use in `gatsby-browser.js`. If set to `false`, the plugin does not inject styles (i.e. `StylesProvider` is not wrapped around `element`, in `wrapRootElement`. I included the option because @hupe1980 originally had such a check based on whether or not the `stylesProvider` option was given in `gatsby-config.js`.

Two alternatives:

1) We can take the option out, and assume that if people provide the `pathToConfigModule` then the plugin automatically injects the styles. If no such option is given, the styles are not injected. This would be similar to how the original implementation by @hupe1980 works

2) We can remove the check altogether, for whether or not the injection should happen. This way we assume that if someone uses the plugin, then he/she wants the `StyleProvider` to be wrapped in `wrapRootElement`

PS: The README has some info.